lib/sync_checker/formats/edition_base.rb

```diff
@@ -161,7 +161,7 @@ def rendering_app
         Whitehall::RenderingApp::WHITEHALL_FRONTEND
       end
 
-      def document_type
+      def document_type(_edition = nil)
```

Comment:
`DetailedGuideCheck` also implements this without any arguments so that will need changing/removing. I think it only gets called in this file on line 150 too so probably doesn't need to be optional.